### ISSUE TITLE
Apps not distributed due to missing contact email

### ISSUE THREAD

**Issue by ayanyev** [guness/bitrise-step-firebase-app-distribution#24]
Distribution worked until certain point of time. After appeared a weird error: missing contact email. But I am sure that there is a contact email for each distributed app set - it cannot be not set

> 
| (2) Firebase deploy - dev                                                    |
+------------------------------------------------------------------------------+
| id: firebase-app-distribution                                                |
| version: 0.6.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: bash                                                                |
| time: 2020-04-07T15:48:52Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
Configs:
  * firebase_token: [REDACTED]
  * service_credentials_file: 
  * app_path: /bitrise/deploy/app-development-debug.apk
  * app: 1:<...>
  * release_notes: 
  * release_notes_file: 
  * testers: 
  * groups: testers
  * flags: 
  * is_debug: true
App path contains one file 👍
-- Checking for existing firebase-tools on PATH...
error: unknown option '--tool:setup-check'
Your machine has firebase-tools@7.15.1 installed.
If you would like to upgrade your install run: curl -sL firebase.tools | upgrade=true bash
Deploying build to Firebase
  firebase appdistribution:distribute '/bitrise/deploy/app-development-debug.apk' --app '1:<...>' --groups 'testers' --debug
[2020-04-07T15:48:59.681Z] ----------------------------------------------------------------------
[2020-04-07T15:48:59.684Z] Command:       /usr/bin/node /usr/bin/firebase appdistribution:distribute /bitrise/deploy/app-development-debug.apk --app 1:<...> --groups testers --debug
[2020-04-07T15:48:59.685Z] CLI Version:   7.15.1
[2020-04-07T15:48:59.685Z] Platform:      linux
[2020-04-07T15:48:59.685Z] Node Version:  v10.19.0
[2020-04-07T15:48:59.685Z] Time:          Tue Apr 07 2020 15:48:59 GMT+0000 (Coordinated Universal Time)
[2020-04-07T15:48:59.685Z] Env Overrides: FIREBASE_APP_DISTRIBUTION_URL
[2020-04-07T15:48:59.685Z] ----------------------------------------------------------------------
[2020-04-07T15:48:59.686Z] 
[2020-04-07T15:48:59.694Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[2020-04-07T15:48:59.694Z] > authorizing via FIREBASE_TOKEN environment variable
i  getting app details...
[2020-04-07T15:48:59.697Z] > refreshing access token with scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[2020-04-07T15:48:59.698Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[2020-04-07T15:48:59.789Z] <<< HTTP RESPONSE 200 content-type=application/json; charset=utf-8, vary=X-Origin, Referer, Origin,Accept-Encoding, date=Tue, 07 Apr 2020 15:48:59 GMT, server=scaffolding on HTTPServer2, cache-control=private, x-xss-protection=0, x-frame-options=SAMEORIGIN, x-content-type-options=nosniff, accept-ranges=none, transfer-encoding=chunked
[2020-04-07T15:48:59.791Z] >>> HTTP REQUEST GET https://console.firebase.google.com/u/0/project/<...>/appdistribution/app/android/<...>/releases/v1alpha/apps/1:<...>
 
[2020-04-07T15:49:00.173Z] <<< HTTP RESPONSE 200 content-type=text/html; charset=utf-8, cache-control=no-cache, no-store, max-age=0, must-revalidate, pragma=no-cache, expires=Mon, 01 Jan 1990 00:00:00 GMT, date=Tue, 07 Apr 2020 15:49:00 GMT, p3p=CP="This is not a P3P policy! See g.co/p3phelp for more info.", content-security-policy=script-src 'report-sample' 'nonce-1vf14+yLqG2AhS1D3uIt1w' 'unsafe-inline' 'strict-dynamic' https: http: 'unsafe-eval';object-src 'none';base-uri 'self';report-uri /u/0/cspreport, server=ESF, x-xss-protection=0, x-frame-options=SAMEORIGIN, x-content-type-options=nosniff, set-cookie=[NID=201=Wh6z4t8eV6WpJQ8ugEx2y08xKEYj4RE7KHfMoQuC6fya_edElHUO_ZsjJok5-dR8OvgO6YR7QJLPu4HJRs5LlWuj4Sg7jZXlGfYU1V8Z_SM0t5D5X174hm1LCGpN-vazGCW3dKJGwaZf3eI6B7aRSUDcbgQAYxT2pnz43SJSvLs; expires=Wed, 07-Oct-2020 15:49:00 GMT; path=/; domain=.google.com; HttpOnly], accept-ranges=none, vary=Accept-Encoding, transfer-encoding=chunked
Error: We could not find a contact email for app 1:<...>. Please visit App Distribution within the Firebase Console to set one up.

**Comment by guness** [comment]
@ayanyev the last bug we have fixed (not deployed yet) is related to single quotes on release notes. can you also confirm it is not related to that by removing all release notes input.
?

Also it is good to confirm email address exists on App Distribution console.

**Comment by guness** [comment]
@ayanyev can you retry with the last version (0.7.0)

**Comment by ayanyev** [comment]
Hi. Sorry for the late answer. The situation was really strange. I also tried distributing with Firebase CLI  from Bitrise machine (script step). But failed as well. From local machine apps where distributed without problems
So the problem was not with your Firebase distribution step.
After all I switched to distribution by [Gradle plugin](https://firebase.google.com/docs/app-distribution/android/distribute-gradle).
